kairon/chat/routers/web_client.py

```
@@ -135,6 +135,29 @@ async def execute_flow(
     }
 
 
+@router.post('/chat/exec/flow/media', response_model=Response)
+async def execute_flow(
```

Ruff (0.11.9)

Redefinition of unused `execute_flow` from line 119

(F811)